utils/wasm-builder/tests/smoke.rs

```diff
@@ -153,3 +153,19 @@ fn features_tracking() {
     assert!(read_export_entry("handle_signal").is_some());
     assert!(read_export_entry("handle_reply").is_none());
 }
+
+#[ignore]
```

Comment:
why ignored?

Reply:
cause it doesn't guarantee in future versions that syn crate will be duplicated

and it takes more than a minute to run